examples_tests/__init__.py

```diff
@@ -125,8 +125,8 @@ def build_snap(self, example_dir):
             [self.snapcraft_command, 'clean'], cwd=working_dir)
         try:
             subprocess.check_output(
-                [self.snapcraft_command, 'snap'], cwd=working_dir,
-                stderr=subprocess.STDOUT).decode('utf-8')
+                [self.snapcraft_command, '--debug', 'snap'], cwd=working_dir,
+                stderr=subprocess.STDOUT)
```

Comment:
@elopio any idea what is going on here?

@elopio I'm lost as with what is going on here:

```
Traceback (most recent call last):
  File "/home/jenkins-slave/workspace/github-snapcraft-examples-tests-cloud/examples_tests/tests.py", line 98, in test_example
    self.build_snap(self.example_dir)
  File "/home/jenkins-slave/workspace/github-snapcraft-examples-tests-cloud/examples_tests/__init__.py", line 129, in build_snap
    stderr=subprocess.STDOUT).decode('utf-8')
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
```

Is jenkins running `python3 setup.py develop` by any chance?